### PR TITLE
Changes to the experimentor outpost:

### PR DESCRIPTION
**Pull request by Deepsloth**
- Starlanes will be created whenever monsters are spawned (not the turn later).
- The experimentor building won't destroy itself after being conquered and will go on creating starlanes and spawning monsters for the owner.

### CODE REVIEW

File: default/buildings.txt [hunk 1]
```diff
@@ -1730,31 +1736,22 @@ BuildingType
                     WithinDistance distance = 100 condition = Source
                     System
                 ]
-                Destroy
```
Comment (MatGB) [inline default/buildings.txt:1733]: Really unsure about letting it stay in place after conquest and creating beasties for the player. You sometimes get a few on the turn it destroys itself and they're hideous if they're under sentient control.

Also, won't the whole 'not destroying' thing mean the Sitrep is triggered every turn? It's bad enough that Transcendance does that.

**Comment by Dilvish-fo** [comment]
I agree the experimentor mobs seem OTT to be under player control.  

After the Outpost is under empire ownership, the make/delete starlane thing could just be random, rather than triggered by monsters.

As for sitreps (here or for Transcendance), the change of ownership (or tech for transcendance) could apply a special to the Capital, which has a very low Capacity counting down each turn, and it only sends sitreps while the capacity is above zero.

**Comment by Deepsloth** [comment]
As discussed (here: http://www.freeorion.org/forum/viewtopic.php?f=28&t=9585), i reduced this PR to just fix the starlane issues.

**Comment by Dilvish-fo** [comment]
Have you confirmed this with testing, or is it an 'expected to work' proposal? 

Also, something to think about is whether in some galaxies the 100 uu distance in 

```
WithinDistance distance = 100 condition = Source
```

might be too short and perhaps that is what caused the observed problem that prompted this recent attention.  Perhaps instead of 100 we should have some like the max of 100 or 1.5 times the distance to the nearest system

**Comment by Deepsloth** [comment]
Since all starlane creations of the experimentor outpost have a distance of 100 uu, it's possible that the outpost never connects and monsters will be stranded, but that has nothing to do with the reported problems.

As i've posted in the forums, i'm confident that the problem stems from the processing of the scripts, i.e. a starlane gets created by one effect and is removed by another effect even though the effects are ordered differently in the FOCS file. This is still happening: sometimes when new monsters are spawned no new starlane appears. And i've even seen that the monsters got a move order (placed on the upper left of the system) without there being any starlanes on that turn or the turn before.

I did test the new fix for the starlanes creation after conquest and it always worked, but maybe there can still be corner cases where all 5 starlanes are created on the same spot and that single starlane is removed again. The chance of this happening should be greatly reduced.
